pkgs/applications/editors/mg/default.nix

```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
   '';
   nativeBuildInputs = [ pkg-config ];
 
-  buildInputs = [ ncurses ];
+  buildInputs = [ ncurses ] ++ lib.optional stdenv.isDarwin [ darwin.libutilHeaders ];
```

Comment:
What's the difference between `libutilHeaders` and `libutil`? For me, this builds with just `libutil` and switching between the two doesn't seem to rebuild.

Reply:
You are right, I mixed up the two. Looks like `libutilHeaders` is just a compatibility alias: https://github.com/NixOS/nixpkgs/commit/6c6b7b0cc018ee0a6cf44d572f0a2b26c4b4a171#diff-3a42b6a0e18465380f92bfca2e33c869d7fcc7e96fec8c0ecf768e05fdeae210

I updated my PR with `darwin.libutil` everywhere.